tests/quality_pragmaitc/test_refinement.py

```diff
@@ -24,50 +29,51 @@ def test_shouldBeApplicable():
 
     fullContext.append(contextCurrent)
 
-    #goal.addApplicableContext(contextCurrent)
+    goal.addApplicableContext(contextCurrent)
     task.addApplicableContext(contextCurrent)
     delegation.addApplicableContext(contextCurrent)
 
-    #assert True == goal.isApplicable(fullContext)
-    assert True == task.isApplicable(fullContext)
-    assert True == delegation.isApplicable(fullContext)
+    assert True is goal.isApplicable(fullContext)
+    assert True is task.isApplicable(fullContext)
+    assert True is delegation.isApplicable(fullContext)
 
     def test_shouldBeNotApplicable():
-        #goal = Goal()
+        goal = Goal()
         task = Task()
         delegation = Delegation()
 
         context = Context("C1")
 
         task.addApplicableContext(context)
-        #goal.addApplicableContext(context)
+        goal.addApplicableContext(context)
         delegation.addApplicableContext(context)
 
         wrongContext = Context("C2")
         fullContext = []
-        fullContext.add(wrongContext)
+        fullContext.append(wrongContext)
 
-        #assert False == goal.isApplicable(fullContext)
-        assert False == task.isApplicable(fullContext)
-        assert False == delegation.isApplicable(fullContext)
+        assert False is goal.isApplicable(fullContext)
+        assert False is task.isApplicable(fullContext)
+        assert False is delegation.isApplicable(fullContext)
 
     def test_aTaskShouldBeAchievable():
-        commonMetrics = CommonMetrics()
         task = Task()
 
         currentContext = Context("C1")
         fullContext = []
         fullContext.append(currentContext)
 
-        qc = QualityConstraint(currentContext, commonMetrics.SECONDS, 15, 'LESS_OR_EQUAL_TO')
+        qc = QualityConstraint(
+            currentContext, CommonMetrics.SECONDS, 15, 'LESS_OR_EQUAL_TO')
 
         task.addApplicableContext(currentContext)
-        task.setProvidedQuality(currentContext, commonMetrics.SECONDS, 12)
+        task.setProvidedQuality(currentContext, CommonMetrics.SECONDS, 12)
 
         interp = Interpretation()
         interp.addQualityConstraint(qc)
 
-        assert True == task in task.isAchievable(fullContext, interp).getTasks()
+        assert True is task in task.isAchievable(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)